src/blocks/advanced-heading/edit.js

```diff
@@ -24,6 +24,7 @@ const UAGBAdvancedHeading = ( props ) => {
 		clientId,
 		name,
 		deviceType,
+		context
```

Comment:
Always add trailing comma.

```suggestion
		context,
```

Reply:
This coding practice has been implemented [in this PR](https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/3283).

This conversation can be resolved once that PR is merged.